test/e2e/service.go

```diff
@@ -797,6 +797,75 @@ var _ = Describe("Services", func() {
 	})
 })
 
+var _ = Describe("Services", func() {
+	f := NewFramework("services-temp")
+
+	var t *explorerTest
+	BeforeEach(func() {
+		t = newDefaultExplorerTest(f.Client, f.Namespace.Name)
+	})
+
+	It("should serve round-robin services round-robin", func() {
+		_, _, svc := t.setup()
+
+		nodeName := getFirstNodeNameOrDie(f.Client)
+
+		h := histogram{}
```

Comment:
This is contrary to https://github.com/golang/go/wiki/CodeReviewComments#variable-names